<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PWMBuildDialog</class>
 <widget class="QDialog" name="PWMBuildDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build Weight or Frequency Matrix</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Input file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="inputButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="outputEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="outputButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="statisticsBox">
     <property name="title">
      <string>Statistic options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="statisticsLabel">
        <property name="text">
         <string>Statistic type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mononucleicButton">
        <property name="text">
         <string>Mononucleic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dinucleicButton">
        <property name="text">
         <string>Dinucleic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="matrixBox">
     <property name="title">
      <string>Matrix options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="matrixLabel">
          <property name="text">
           <string>Matrix type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="frequencyButton">
          <property name="text">
           <string>Frequency matrix</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="weightButton">
          <property name="text">
           <string>Weight matrix</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,2">
        <item>
         <widget class="QLabel" name="algorithmLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Weight algorithm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="algorithmCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="logoWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>weightButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>algorithmLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weightButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>algorithmCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PWMBuildDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
